elementary/monitor/api/invocations/invocations.py

```diff
@@ -37,3 +39,11 @@ def get_invocation_by_id(
             )
         else:
             raise NotImplementedError
+
+    def get_invocations_by_ids(self, invocations_ids: list[str]) -> DbtInvocationSchema:
```

Comment:
```suggestion
    def get_invocations_by_ids(self, invocations_ids: list[str]) -> List[DbtInvocationSchema]:
```

Reply:
Why? Using List is deprecated
https://peps.python.org/pep-0585/#implementation

```
list # typing.List

dict # typing.Dict

...

Importing those from typing is deprecated. Due to PEP 563 and the intention to minimize the runtime impact of typing, this deprecation will not generate DeprecationWarnings. Instead, type checkers may warn about such deprecated usage when the target version of the checked program is signalled to be Python 3.9 or newer. It's recommended to allow for those warnings to be silenced on a project-wide basis.

The deprecated functionality will be removed from the typing module in the first Python version released 5 years after the release of Python 3.9.0.
```